### PR TITLE
Fix spelling/grammar errors in components

### PR DESCRIPTION
**Pull request by adrianwright109**
Fix various spelling/grammar errors in components.

### CODE REVIEW

**Comment by adrianwright109** [comment]
@HaoK Any idea why **Tests: Helix ARM64** failed?
Logs here: https://github.com/aspnet/AspNetCore/pull/13953/checks?check_run_id=221660157

**Comment by HaoK** [comment]
No, we just enabled these tests today, and it looks like we don't have the proper setting that flows the secret needed for forks to properly run these, I'm just waiting for confirmation that its safe for us to enable this setting.

**Comment by HaoK** [comment]
Closing and reopening to force a rebuild

**Review by moslem-hadi — APPROVED** [review]
Its just grammer check! 👍

File: src/Components/Components/src/Rendering/Renderer.cs [hunk 1]
```diff
@@ -139,7 +139,7 @@ protected async Task RenderRootComponentAsync(int componentId, ParameterView ini
             // remaining work.
             // During the synchronous rendering process we don't wait for the pending asynchronous
             // work to finish as it will simply trigger new renders that will be handled afterwards.
-            // During the asynchronous rendering process we want to wait up untill al components have
+            // During the asynchronous rendering process we want to wait up until al components have
```
Comment (mkArtakMSFT) [inline src/Components/Components/src/Rendering/Renderer.cs]: can you also fix the `al` to `all`

**Comment by mkArtakMSFT** [comment]
Thanks for your effort, @adrianwright109.
It seems there is something wrong with how you rebased your changes, as too many commits show up in here, which are irrelevant. Can you please clean this up, so only your own changes show up?

**Comment by adrianwright109** [comment]
I have created a cleaner PR #14137 which superceeds this PR - closing the PR in favour of PR #14137

**Pull request closed without merging**